language/move-stdlib/sources/FixedPoint32.move

```diff
@@ -173,12 +173,12 @@ module Std::FixedPoint32 {
     }
     spec create_from_rational {
         pragma opaque;
-        include [concrete] ConcreteCreateFromRationalAbortsIf;
-        ensures [concrete] result == spec_concrete_create_from_rational(numerator, denominator);
+        include [concrete] CreateFromRationalAbortsIf;
+        ensures [concrete] result == spec_create_from_rational(numerator, denominator);
         include [abstract] CreateFromRationalAbortsIf;
         ensures [abstract] result == spec_create_from_rational(numerator, denominator);
```

Comment:
Maybe we can remove the `[concrete]` and `[abstract]` such that we unify both?

Reply:
I merged them.